pyomo/contrib/mindtpy/tests/eight_process_problem.py

```diff
@@ -32,7 +32,7 @@
 class EightProcessFlowsheet(ConcreteModel):
     """Flowsheet for the 8 process problem."""
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, convex=True, *args, **kwargs):
```

Comment:
I don't know if this function works. But I would suggest putting the default arguments behind the non-default arguments.
FYI, https://stackoverflow.com/questions/16932825/why-cant-non-default-arguments-follow-default-arguments